tidy3d/components/grid/grid_spec.py

```diff
@@ -572,6 +585,15 @@ class GridSpec(Tidy3dBaseModel):
         "uses :class:`.AutoGrid`.",
     )
 
+    fixpoints: Tuple[Point, ...] = pd.Field(
```

Comment:
I'm wondering if these should be `fixed_points` rather than `fixpoints`. Is "fixpoints" commonly used?

Reply:
It's used in one software. In fact, I'm not so in favor of this name. How about other names such as `snapping_points`, `enforced_grid_points`? @tylerflex @QimingFlex

Reply:
Rename to `snapping_points` now.